staging/src/k8s.io/api/policy/v1beta1/types.go

```diff
@@ -60,7 +60,8 @@ type PodDisruptionBudgetStatus struct {
 	// the list automatically by PodDisruptionBudget controller after some time.
 	// If everything goes smooth this map should be empty for the most of the time.
 	// Large number of entries in the map may indicate problems with pod deletions.
-	DisruptedPods map[string]metav1.Time `json:"disruptedPods" protobuf:"bytes,2,rep,name=disruptedPods"`
+	// +optional
+	DisruptedPods map[string]metav1.Time `json:"disruptedPods,omitempty" protobuf:"bytes,2,rep,name=disruptedPods"`
```

Comment:
You can mark it `+optional`, but you can't mark it omitempty once a field has gone to beta (you might break a client that expects to always get a field back in json).  So remove the omitempty part.

Reply:
That makes sense. Updated